src/components/EventList/components/EventItem/index.jsx

```diff
@@ -32,6 +36,18 @@ const EventItem = ({ event }) => {
           <Typography type="caption" className={styles.descriptionText}>
             {event.Description || 'No description available'}
           </Typography>
+          <Button onClick={() => alert(event.date +" " +event.timeRange)}
+          >name</Button>
```

Comment:
Since this PR is open now, you shouldn't have any testing code left behind in it.

Reply:
Thanks for reminder